lib/react/README.md

```diff
@@ -0,0 +1,60 @@
+# GitHub Octicons React Component
+
+[![npm version](https://img.shields.io/npm/v/%40github%2Focticons-react.svg)](https://www.npmjs.org/package/%40github%2Focticons-react)
+[![Build Status](https://travis-ci.org/primer/octicons.svg?branch=master)](https://travis-ci.org/primer/octicons)
+
+> A react component for installing octicons
+
+## Install
+
+```
+$ npm install @github/octicons-react --save
+```
+
+## Usage
+
+The entire library will be available when importing `@github/octicons-react`. Specifying the [icon you want to use][octicons], by supplying the `name=""` to the component.
+
+```js
+// Example usage
+import Octicon from "@github/octicons-react"
+
+const AlertIcon = <Octicon name="alert" />
```

Comment:
I think showing examples where you use the icons within another other component might be clearer, this is example is kind of making a component of a component 🙃

Here's a quick example:
```js
<ButtonDanger>
  <Octicon name='alert'>
  Delete
</ButtonDanger>
```

Also, pretty sure we want to be using single quotes.